doc/architecture/architecture.rst

```diff
@@ -1,24 +1,254 @@
+.. _`meta-soletta`: https://github.com/solettaproject/meta-soletta.git
+.. _`meta-swupd`: http://git.yoctoproject.org/meta-swupd
+
+============================ =======================================
+Layer Repository Name        Description
+============================ =======================================
+`openembedded-core`_         Core metadata and component recipes
+`meta-intel`_                BSP layer for common IA platforms
+`meta-ostro`_                Ostro OS distro metadata, configuration,
+                             and documentation
+`meta-ostro-fixes`_          Ostro OS layer that is used to carry
+                             fixes to upstream layers
+`meta-ostro-bsp`_            Ostro OS BSP configuration metadata
+`meta-edison-bsp`_           BSP layer for Intel Edison (integrated
+                             via the `meta-ostro-bsp`_ layer)
+`meta-intel-iot-security`_   Security building blocks: IMA, SMACK.
+`meta-appfw`_                Ostro OS application framework recipes
+                             and sample applications
+`meta-openembedded`_         Collection of OpenEmbedded layers
+`meta-oic`_                  `IoTivity`_
+`meta-intel-iot-middleware`_ Middleware components used in Intel IoT
+                             DevKit
+`meta-iotqa`_                Ostro OS test tools
+`meta-iot-web`_              Node.JS and OIC JavaScript APIs
+`meta-security-isafw`_       Image Security Analysis framework gives
+                             offline tooling to analyze images
+`meta-yocto`_                BSP layer for BeagleBone Black
+`meta-java`_                 Java support (openjdk8)
+`meta-soletta`_              `Soletta Project`_
+`meta-swupd`_                Software update tooling
+============================ =======================================
+
+The Ostro OS distro configuration file (:file:`meta-ostro/conf/distro/ostro.conf`) defines
+the global configuration of the OS. For example, it determines which components are being
+enabled/built by default (via ``DISTRO_FEATURES``, ``PNBLACKLIST``, and ``PNWHITELIST`` settings).
+
+Ostro OS Development Workflows
+==============================
+
+.. _`Yocto Project extensible SDK`: http://www.yoctoproject.org/docs/2.1/sdk-manual/sdk-manual.html#sdk-extensible
+
+Ostro OS enables many options how to do hacking and/or IoT product development.
+
+#. **Quick Prototyping**: Using the pre-built development images found in `Ostro Project download server`_ it's easy to get started with any of the :ref:`platforms`. Some of the image configurations also provide an environment to do quick prototyping with programming tools discussed in 'Programming Interfaces' above.
+#. **Image Customization**: If the pre-built development images do not provide the desired content or configuration, images can easily be customized to suit the needs. Alternatively, :ref:`software-update` could be used to extend the image content by installing additional bundles.
```

Comment:
Rephrase this thinking about products. There is no "if", products will customize the images in a way :-)

Remove the bundles mention, confuses the reader about the customizations